medusa/trakt_checker.py

```diff
@@ -206,19 +206,21 @@ def sync_library(self):
                 logger.log(u"Synced Trakt collection", logger.DEBUG)
 
     def remove_episode_trakt_collection(self, filter_show=None):
-        """Remove episode from trakt collection."""
+        """Remove episode from trakt collection that no longer has media file (location)."""
```

Comment:
You don't need to explain the method in detail in your summary. If you really want to explain conditions in your method, then provide that on a separate line, with one white line in between. Or just add a comment above the condition.